### ISSUE TITLE
Add support for block strings

### ISSUE THREAD

**Issue by frasercobb** [graph-gophers/graphql-go#298]
Hello,
This is a feature request to support "Block Strings" as per the spec: https://facebook.github.io/graphql/June2018/#sec-String-Value

I have reproduced this issue in the starwars example in this repo.

**Example mutation that will trigger this issue:**
```
mutation {
  createReview(episode:NEWHOPE, review: {stars: 0, commentary: """1
    2"""}) {
    stars
  }
}
```
**Response:**
```
{
  "errors": [
    {
      "message": "syntax error: unexpected \"\\\"1\\n\", expecting Ident",
      "locations": [
        {
          "line": 31,
          "column": 66
        }
      ]
    }
  ]
}
```
**Log output:** `<input>:31:66: literal not terminated `

**Comment by pavelnikolov** [comment]
Thanks for reporting!

**Comment by dackroyd** [comment]
I believe this has since been resolved by #269 ? Requires an _opt-in_ option (`UseStringDescriptions()`) to be applied when creating the schema.

**Comment by pavelnikolov** [comment]
You are right, Dave. I'm closing this issue.
